Resources/Private/JavaScript/Guest/Containers/NodeToolbar/Buttons/AddNode/index.js

```diff
@@ -4,8 +4,12 @@ import {
     Icon
 } from 'Components/index';
 
+// TODO: hackish! What is the proper way to call `ui` methods?
+const {ui} = window.neos;
```

Comment:
How to do this properly? What are signals and should they be used instead? Or should I fire a guest redux action and call `ui.openAddNodeModal` from Saga? Help, I have not clue!